src/Tizen.NUI/src/public/XamlBinding/Element.cs

```diff
@@ -57,7 +57,7 @@ public abstract partial class Element : BindableObject, IElement, INameScope, IE
         /// </summary>
         /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public string AutomationId
+        public virtual string AutomationId
```

Comment:
How about change as followings
public override string AutomationId => public new string AutomationId
public virtual string AutomationId => public string AutomationId (no change)
(personally, Element is base class and it looks not good to change base class's interface)